lib/services/elasticsearch.js

```diff
@@ -1153,7 +1153,21 @@ class ElasticSearch extends Service {
       }
     }
 
-    return this._mexecute(esRequest, toImport, extracted.rejected);
+    return this._mexecute(esRequest, toImport, extracted.rejected)
+      .then(response => {
+        // with _source: true, ES returns the updated document in
+        // response.result.get._source
+        // => we replace response.result._source with it so that the notifier
+        // module can seamlessly process all kind of m* response*
+        let j; // NOSONAR
+        for (j = 0; j < response.result.length; j++) {
+          if (response.result[j].get && response.result[j].get._source) {
```

Comment:
It's a bit hard to read, why not use a variable ? `const res = response.result[j];`

Reply:
good point

Reply:
should be easier to read now